lib/broadside/cli/gli_commands.rb

```diff
@@ -0,0 +1,142 @@
+def add_tag_flag(cmd)
```

Comment:
i'm also going to go on record as liking (some of the) metaprogramming - it kept you from typing the same thing over and over.  `.public_send(:symbol)`  is a really powerful thing when you are mapping CLI input to actual method calls.  how much did you dislike it?

(p.s. i'm exhausted and might need to revisit what this change really did tomorrow, but i still think i like the metaprogramming you had from the beginning - at least the `send(:method)` part)